iocs/PICamIOC/iocBoot/iocPICam/st.cmd

```diff
@@ -12,6 +12,8 @@ epicsEnvSet("XSIZE",  "2048")
 epicsEnvSet("YSIZE",  "2048")
 epicsEnvSet("NCHANS", "2048")
 # The search path for database files
+epicsEnvSet("ADCORE", "/home/towa/EPICS/support/areaDetector/ADCore")
+epicsEnvSet("ADPICAM","/home/towa/EPICS/support/areaDetector/ADPICam")
```

Comment:
These additions of ADCORE and ADPICAM are not needed.  These will automatically be defined in envPaths.